app/hotels/src/singleHotel/roomList/RoomDescription.js

```diff
@@ -28,16 +32,26 @@ const styles = StyleSheet.create({
     fontSize: 12,
     color: '#30363d',
   },
+  readMore: {
+    color: Color.brand,
+    fontWeight: '600',
+    fontSize: 12,
+  },
 });
 
-const RoomDescription = (props: Props) => {
+export const RoomDescription = (props: Props) => {
   const description = idx(props, _ => _.room.description.text);
 
   if (description != null) {
     return (
       <View>
         <View style={styles.delimiter} />
-        <ReadMore numberOfLines={2}>
+        <ReadMore
+          numberOfLines={2}
+          linkStyle={styles.readMore}
```

Comment:
Can this be just `style` as usual? I know there is also view around but you cannot style it from outside anyway.